.eslintrc.js

```diff
@@ -6,16 +6,16 @@ module.exports = {
   },
   extends: [
     '@nuxtjs/eslint-config-typescript',
-    'plugin:nuxt/recommended'
+    'plugin:vitest/recommended'
   ],
-  plugins: ['vitest'],
   rules: {
-    semi: ['warn', 'always'],
+    semi: ['error', 'always'],
     quotes: ['warn', 'single'],
     indent: ['warn', 2],
     'max-len': ['warn', { code: 120 }],
-    '@typescript-eslint/no-unused-vars': 'off',
     'vue/multi-word-component-names': 'off',
-    'vue/no-multiple-template-root': 'off'
+    'vue/component-tags-order': ['error', {
+      order: ['script:not([setup])', 'script[setup]', 'template', 'style']
+    }]
```

Comment:
I am not sure about pushing the order of the component since this is a open source and we are just providing some base for anyone.

Reply:
While I (personally) prefer this order and it is widely used in Vue doc examples, I see your point. Let's ask for @bastian-friedrich opinion. He recently asked me to use this rule in other project and liked its idea in general.

Reply:
tbh, I like that this is pre-defined so there are no questions how to do this and all devs using the same order. And the best: eslint will also format it. I highly like it! And if people dont like it, they can easily delete the rule. Let's go for it.

Reply:
@bastian-friedrich thanks for your opinion!